samza-api/src/main/java/org/apache/samza/operators/functions/InputTransformer.java

```diff
@@ -25,7 +25,7 @@
  * Transforms an {@link IncomingMessageEnvelope} with deserialized key and message to a message of type {@code OM}
  * which is delivered to the {@code MessageStream}.
  * <p>
- * This is provided by default by {@code TransformingSystemDescriptor} implementations and can not be overridden
+ * This is provided by default by {@code TransformingInputDescriptorProvider} implementations and can not be overridden
```

Comment:
This comment seems a little inconsistent now, since system descriptors don't have to implement this class in order to provide a transformer.